reana/reana_dev/release.py

```diff
@@ -189,9 +190,12 @@ def release_pypi(ctx, component, timeout):  # noqa: D301
 
 
 @click.option("--user", "-u", default="reanahub", help="DockerHub user name [reanahub]")
+@click.option(
+    "--dry-run", is_flag=True, default=False, help="Do not execute any actions."
```

Comment:
Minor: perhaps `Display the command without executing it.` would be more informative?

Reply:
I agree, it's more clear. I copied it from https://gitlab.cern.ch/reanahub/deployment/-/blob/master/deployment.py#L53 so we might want to change it there also.